lib/spack/docs/getting_started.rst

```diff
@@ -24,6 +24,7 @@ before Spack is run:
 #. The ``patch`` command to apply patches
 #. The ``git`` and ``curl`` commands for fetching
 #. If using the ``gpg`` subcommand, ``gnupg2`` is required
+#. If using the ``unit-test`` subcommand, ``file`` is required
```

Comment:
`file` is also required for buildcaches, both to create or restore them iirc

Reply:
Also, not sure exactly when it will land, but I turned this part of the docs into a proper table and added a few other explicit dependencies here (in case you want to review that commit): https://github.com/spack/spack/pull/24003/commits/861e474c226e881bf8d36f8bc6d02fff3fe0f99d The file of interest is the `.csv` file in that commit.

Reply:
Sure, the PR in which docs are changed in this sense is #22720 (which is much broader in scope). Search for the `.csv` file, which has data for the table, and the `.rst` file in the changeset